src/utils/version.js

```diff
@@ -48,7 +48,7 @@ const version = {
     // Set network in lib to use the correct address byte
     let network = data.network;
 
-    if (data.network.startsWith('testnet')) {
+    if (data.network.includes('testnet')) {
```

Comment:
This is also happens in other files, like: 

- `src/components/ServerStatus.js#33` 
- `src/screens/Server.js#L240`

Reply:
thought(non-blocking): I have the impression that this is another util that would be helpful to have centralized on the lib itself. We recently had some issues with this specific code on the mobile wallet as well.

Reply:
I created a new PR in the lib for this (https://github.com/HathorNetwork/hathor-wallet-lib/pull/696) because this was also a bug in the mobile.

As soon this PR is merged and the lib version is released, I will create a PR to use this method in both wallets.